ui/app/components/app/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -112,6 +110,8 @@ const mapStateToProps = (state, ownProps) => {
   const isMainnet = getIsMainnet(state)
   const showFiat = Boolean(isMainnet || showFiatInTestnets)
 
+  const hideBasic = state.appState.modal.modalState.props.hideBasic || (!isMainnet && !process.env.IN_TEST)
```

Comment:
Unfortunately the gas chart shown on the 'Advanced' tab is _also_ built using information from Eth Gas Station, so hiding the basic tab isn't especially helpful. It's the 'Edit' button on the transaction confirmation screen that leads here that needs to be hidden:

![advanced-gas-confirm](https://user-images.githubusercontent.com/2459287/90556931-11d54e00-e170-11ea-9087-42bca0c2bb2c.png)